CMakeLists.txt

```diff
@@ -25,6 +29,7 @@ option(MZ_FORCE_FETCH_LIBS "Enables fetching third-party libraries always" OFF)
 option(MZ_PKCRYPT "Enables PKWARE traditional encryption" ON)
 option(MZ_WZAES "Enables WinZIP AES encryption" ON)
 option(MZ_OPENSSL "Enables OpenSSL for encryption" ${UNIX})
+option(MZ_BCRYPT "Enables BCrypt for encryption" ${WIN32UWP})
```

Comment:
```cmake
option(MZ_BCRYPT "Enables BCrypt for encryption" (CMAKE_SYSTEM_NAME STREQUAL "WindowsStore"))
```

Reply:
`MZ_BCRYPT` also needs to be documented in the `README.md`.

Reply:
> Does this work?

IDK, need try it.

Reply:
Made it a dependent option, forced ON on WindowsStore and user choice otherwise.